demo/deploy-kafka.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+set -aueo pipefail
+
+# shellcheck disable=SC1091
+source .env
+DEPLOY_ON_OPENSHIFT="${DEPLOY_ON_OPENSHIFT:-false}"
+USE_PRIVATE_REGISTRY="${USE_PRIVATE_REGISTRY:-false}"
+MESH_NAME="${MESH_NAME:-osm}"
+
+kubectl create ns kafka 
+
+bin/osm namespace add --mesh-name "$MESH_NAME" kafka
+
+bin/osm metrics enable --namespace kafka
+
+helm install kafka bitnami/kafka --set replicaCount=3 --set zookeeper.enabled=false --set zookeeperChrootPath='/kafka-root' --set serviceAccount.create=true --set serviceAccount.name=kafka --namespace kafka --set "externalZookeeper.servers={kafka-zookeeper-0.kafka-zookeeper-headless.zookeeper.svc.cluster.local,kafka-zookeeper-1.kafka-zookeeper-headless.zookeeper.svc.cluster.local,kafka-zookeeper-2.kafka-zookeeper-headless.zookeeper.svc.cluster.local}"
+
+if [ "$DEPLOY_ON_OPENSHIFT" = true ] ; then
+    oc adm policy add-scc-to-user privileged -z "kafka" -n "kafka"
+    if [ "$USE_PRIVATE_REGISTRY" = true ]; then
+        oc secrets link "kafka" "$CTR_REGISTRY_CREDS_NAME" --for=pull -n "kafka"
+    fi
+fi
+
+kubectl apply -nkafka -f - <<EOF
```

Comment:
I think setting the namespace here for `kubectl apply` is redundant, as the namespaces are set in the resource manifests.